### PR TITLE
Update ESLint packages

### PR DESCRIPTION
**Pull request by dbacarel**
Signed-off-by: Daniele Bacarella <daniele.bacarella@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "@typescript-eslint/no-non-null-assertion": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
 
-        "simple-import-sort/sort": "error",
+        "simple-import-sort/imports": "error",
```
Comment (dbacarel) [inline .eslintrc.json:44]: `"simple-import-sort/sort"` was replaced in favor of:
` "simple-import-sort/imports"` and  `"simple-import-sort/exports"`

More info https://github.com/lydell/eslint-plugin-simple-import-sort/blob/v7.0.0/CHANGELOG.md

**Comment by dbacarel** [comment]
retest please

**Comment by dbacarel** [comment]
retest this please

**Comment by dbacarel** [comment]
`eslint`: `7.6.0` -> `7.17.0`
`eslint-config-prettier`: `6.11.0` -> `7.1.0`
`eslint-plugin-prettier`: `3.1.4` -> `3.3.0`
`eslint-plugin-simple-import-sort`: `5.0.3` -> `7.0.0`
`eslint-plugin-standard`: `4.0.1` -> `5.0.0`
`eslint-config-standard-with-typescript`: `5.0.0` -> `19.0.1`
`@typescript-eslint/eslint-plugin`: `3.9.0` -> `4.11.1`
`@typescript-eslint/parser`: `3.9.0` -> `4.11.1`
`eslint-plugin-import`: `2.22.0` -> `2.22.1`

**Review by FraukeF — APPROVED** [review]

**Comment by dbacarel** [comment]
retest this please

**Comment by dbacarel** [comment]
retest this please

**Review by germanz — DISMISSED** [review]

**Comment by dbacarel** [comment]
retest this please

**Comment by dbacarel** [comment]
retest this please

**Review by germanz — APPROVED** [review]

**Pull request merged**